docs/branding.md

```diff
@@ -54,3 +54,30 @@ function add_colour_scheme() {
 	);
 }
 ```
+
+## Custom Favicon
+The favicon shown on the CMS can be set using wither the CMS or Configuration option. Using the CMS option will override the Configuration option.
+
+#### CMS option
+1. Login to the admin area
+2. Go to <strong>Appearance</strong> > <strong>Customize</strong> and click on the <strong>Site Identity</strong> tab
+3. Scroll to <strong>Site Icon</strong> click on the select site icon button and then upload the image you want to use as the icon.
```

Comment:
HTML in markdown is usually a last resort for unsupported or complex markup, use `**` to denote bold text instead.

https://www.markdownguide.org/basic-syntax/